### PR TITLE
feat: upgrade probot to v11.0.1

### PR DESCRIPTION
**Pull request by shine2lay**
Upgrade probot to 11.0.1 and addresses breaking changes
- context.github is now context.octokit
- fix tests

### CODE REVIEW

File: __tests__/e2e/smoke.test.js [hunk 1]
```diff
@@ -13,8 +13,15 @@ describe('smoke tests', () => {
 
   beforeEach(() => {
     nock.disableNetConnect()
-    probot = createProbot({ id: 1, cert: 'test', githubToken: 'test' })
-    probot.load(mergeable)
+    probot = new Probot({
+      githubToken: 'test',
+      // Disable thrttling & retrying requests for easier testing
+      Octokit: ProbotOctokit.defaults({
+        retry: { enabled: false },
```
Comment (jusx) [inline __tests__/e2e/smoke.test.js:20]: Should this be set to `true` ?

Comment (jusx) [reply __tests__/e2e/smoke.test.js:20]: Ah nvm. I see now this is for testing :-)

**Review by jusx — APPROVED** [review]
LGTM!

**Comment by jusx** [comment]
@shine2lay Looks like CI is failing potentially due to package-lock discrepancies.

**Comment by shine2lay** [comment]
@jusx fixed the circle CI issue

**Pull request merged**